### PR TITLE
fix(doclint): correctly get versions on windows

### PR DESCRIPTION
**Pull request by JoelEinbinder**
`--version` doesn't work on windows.

### CODE REVIEW

**Review by aslushnikov — COMMENTED** [review]
Is this the only option?  `winVersionInfo` is [uncomfortably huge](https://npm.anvaka.com/#/view/2d/win-version-info))

**Comment by JoelEinbinder** [comment]
I can launch and parse the user agent. Wdyt?

**Comment by aslushnikov** [comment]
> I can launch and parse the user agent. Wdyt?

SG! Shall we do it on win-only?

**Comment by JoelEinbinder** [comment]
> > I can launch and parse the user agent. Wdyt?
> 
> SG! Shall we do it on win-only?

Uploaded a version that does this. But the firefox version in the user agent is `74.0`, while the one from `--version` is `74.0b10`. I don't know if `b10` means anything to anyone though.

**Review by aslushnikov — APPROVED** [review]

File: utils/doclint/cli.js [hunk 1]
```diff
@@ -103,11 +104,26 @@ async function run() {
...
-    chromium: chromiumVersion.trim().split(' ').pop(),
-    firefox: firefoxVersion.trim().split(' ').pop(),
+    chromium,
+    firefox,
   };
 }
+
+async function getVersion(browserType) {
+  if (os.platform() === 'win32' || os.platform() === 'cygwin') {
+    const browser = await browserType.launch();
```
Comment (aslushnikov) [inline utils/doclint/cli.js]: it's a pity we don't have a `browser.version()`!

**Comment by aslushnikov** [comment]
Oh wait. 

I just understood that `firefox --version` didn't work on win either =/ In this case, it's probably worth just always parse useragent across platforms - otherwise results might be inconsistent (like in firefox case).

Is it fast enough?

**Comment by JoelEinbinder** [comment]
> Oh wait.
> 
> I just understood that `firefox --version` didn't work on win either =/ In this case, it's probably worth just always parse useragent across platforms - otherwise results might be inconsistent (like in firefox case).
> 
> Is it fast enough?

It is very slow, but I can maybe condition it on release?

**Comment by JoelEinbinder** [comment]
I found out that firefox supports `/version` on windows. This gives a big speedup, and gets us back the beta number.

**Review by aslushnikov — APPROVED** [review]

**Pull request merged**